views/profile.handlebars

```diff
@@ -0,0 +1,12 @@
+{{#if loggedIn}}
```

Comment:
Profile View that shows logged in users username and calls partils to load heros